Source/Classes/PINGIFAnimatedImageManager.m

```diff
@@ -462,7 +462,7 @@ + (Float32)frameDurationAtIndex:(NSUInteger)index source:(CGImageSourceRef)sourc
     }
   }
   
-  if (frameDuration < kPINAnimatedImageMinimumDuration) {
+  if (frameDuration < 1.0 / (Float32)[PINAnimatedImage maximumFramesPerSecond]) {
```

Comment:
Is it safe to just cast down an NSInteger to Float32 in this case?

Reply:
I think so, but it's unneccessary I realize as it'll be automatically casted because of the 1.0